Error: Failure("Plugin error")
<<<<<<< HEAD
||||||| 121bedcfd2
Raised by primitive operation at Dynlink.Native.run.(fun) in file "otherlibs/dynlink/native/dynlink.ml", line 87, characters 12-29
Re-raised at Dynlink.Native.run.(fun) in file "otherlibs/dynlink/native/dynlink.ml", line 89, characters 10-149
Called from Stdlib__List.iter in file "list.ml", line 112, characters 12-15
Called from Dynlink_common.Make.load.(fun) in file "otherlibs/dynlink/dynlink_common.ml", line 363, characters 13-56
Called from Stdlib__List.iter in file "list.ml", line 112, characters 12-15
Called from Dynlink_common.Make.load in file "otherlibs/dynlink/dynlink_common.ml", line 359, characters 8-392
Re-raised at Dynlink_common.Make.load in file "otherlibs/dynlink/dynlink_common.ml", line 372, characters 8-17
Called from Dynlink_common.Make.loadfile in file "otherlibs/dynlink/dynlink_common.ml" (inlined), line 374, characters 26-45
Called from Test10_main in file "test10_main.ml", line 49, characters 30-87
=======
Raised by primitive operation at Dynlink.Native.run.(fun) in file "native/dynlink.ml", line 85, characters 12-29
Re-raised at Dynlink.Native.run.(fun) in file "native/dynlink.ml", lines 87-89, characters 10-43
Called from Stdlib__List.iter in file "list.ml", line 112, characters 12-15
Called from Dynlink_common.Make.load.(fun) in file "dynlink_common.ml", line 356, characters 11-54
Called from Stdlib__List.iter in file "list.ml", line 112, characters 12-15
Called from Stdlib__Fun.protect in file "fun.ml", line 34, characters 8-15
Re-raised at Stdlib__Fun.protect in file "fun.ml", line 39, characters 6-52
Called from Dynlink_common.Make.loadfile in file "dynlink_common.ml" (inlined), line 364, characters 26-45
Called from Test10_main in file "test10_main.ml", lines 49-51, characters 30-7
>>>>>>> ocaml/trunk
